test/index.js

```diff
@@ -240,14 +320,22 @@ describe('normalize.undefined', function() {
     expect(result).toEqual(value);
     done();
   });
+
+  it('rejects values that won\'t coerce to undefined', function(done) {
+    var value = 'invalid';
+    var result = normalize.boolean(value);
```

Comment:
Ugh that is a sloppy/paste, indeed. It should have been `normalize.undefined(value)` though, not `.function` (see comment above). I'll fix it and force-push.